conans/test/functional/toolchains/intel/test_inteloneapi.py

```diff
@@ -0,0 +1,99 @@
+import pytest
+import platform
+import textwrap
+
+from conans.test.assets.sources import gen_function_h, gen_function_cpp
+from conans.test.utils.tools import TestClient
+
+
+@pytest.mark.tool_cmake
+@pytest.mark.tool_intel_oneapi
+@pytest.mark.xfail(reason="Intel oneAPI Toolkit is not installed on CI yet")
+@pytest.mark.skipif(platform.system() != "Linux", reason="Only for Linux")
+class TestInteloneAPI:
+
+    @pytest.fixture(autouse=True)
+    def _setUp(self):
+        conanfile = textwrap.dedent("""
```

Comment:
You can get this setup, recommended with ``conan new hello/0.1 -m=cmake_lib`` (or using ``pkg_cmake()`` helper internally should be very close)

Reply:
Yes, I know but I thought it was a better idea to create it from scratch instead of using another internal command. Anyway, it's ok for me 👍

Reply:
``pkg_cmake()`` is actually an internal testing utility designed for this.